### PR TITLE
RED-605 support bondifa recursive resolution

### PR DESCRIPTION
**Pull request by Rednoll**

### CODE REVIEW

File: src/redefined.resolver.ts [hunk 1]
```diff
@@ -57,6 +58,22 @@ export class RedefinedResolver {
             this.resolvers.map(async resolver => {
                 try {
                     const accounts = await resolver.resolve(domain, networks, options);
+
+                    if(resolver.vendor == "redefined-username" || resolver.vendor == "redefined-email") {
```
Comment (au-e2xlabs) [inline src/redefined.resolver.ts:62]: i think we can register "example.sol" only with redefined-username. Therefore, the redefined-email is not needed.

File: tests/unit/redefined.resolver.spec.ts [hunk 2]
```diff
@@ -70,6 +70,96 @@ describe('redefined.resolver', () => {
...
+
+    expect(spyRedefinedEmailResolve).toHaveBeenCalled()
+    expect(spyRedefinedUsernameResolve).toHaveBeenCalled()
+    expect(spySolResolve).toHaveBeenCalledTimes(3)
+  });
+
+  test('SHOULD remove reresolved .sol name from result', async () => {
+    const resolver = new RedefinedResolver({
+      resolvers: [
+        RedefinedResolver.createRedefinedEmailResolver(),
```
Comment (au-e2xlabs) [inline tests/unit/redefined.resolver.spec.ts:95]: usernameResolever

File: src/redefined.resolver.ts (hunk 1 above)
Comment (Rednoll) [reply src/redefined.resolver.ts:62]: it's processing bindings
ETH - 0x12321
SOL - 888.sol

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redefined/name-resolver-js",
-  "version": "2.6.1",
+  "version": "2.7.1",
```
Comment (au-e2xlabs) [inline package.json:3]: 2.7.0

File: src/redefined.resolver.ts (hunk 1 above)
Comment (au-e2xlabs) [reply src/redefined.resolver.ts:62]: ok

**Review by au-e2xlabs — APPROVED** [review]

**Pull request merged**